.github/workflows/create-modeling-round.yaml

```diff
@@ -33,16 +33,19 @@ jobs:
         with:
             python-version-file: "src/.python-version"
 
-      - name: Create clade list 🦠
-        run: |
-          uv run src/get_clades_to_model.py
-
       - name: Set up R 📊
         uses: r-lib/actions/setup-r@v2
 
-      - name: Generate new round config 📝
+      - name: Set up renv 📦
+        uses: r-lib/actions/setup-renv@v2
+        with:
+          working-directory: src
+
+      - name: Create clade list and update tasks.json 🦠
         run: |
-          Rscript src/make_round_config.R
+          uv run get_clades_to_model.py
```

Comment:
Combined the Python and R scripts to run in a single step, with `src/` as the working directory